daml_dit_if/main/log.py

```diff
@@ -42,6 +42,10 @@ def get_log_level():
     return _level
 
 
+def is_debug_enabled():
+    return get_log_level() >= 20
+
+
 def set_log_level(level):
     global _level
```

Comment:
I didn't notice these before, but you could call `setLevel` on the handler to set the log level instead of needing to track it with a global…

Reply:
Which handler? The log level for the integration (modeled in this global) is a single external control that translates into log level settings for four distinct Python loggers. The goal here is to give users a single scalar control for a given integration's log level without exposing them to all the details of the Python logging configuration (that will probably change over time anyway).